fe/fe-core/src/main/java/com/starrocks/sql/ast/CreateCatalogStmt.java

```diff
@@ -27,15 +27,20 @@ public class CreateCatalogStmt extends DdlStmt {
     private final Map<String, String> properties;
     private String catalogType;
 
-    public CreateCatalogStmt(String catalogName, String comment, Map<String, String> properties) {
-        this(catalogName, comment, properties, NodePosition.ZERO);
+    private final boolean ifNotExists;
+
+
+    public CreateCatalogStmt(String catalogName, String comment, Map<String, String> properties, boolean ifNotExists) {
+        this(catalogName, comment, properties, NodePosition.ZERO, ifNotExists);
```

Comment:
better put the NodePosition.ZERO at the last argument

Reply:
got it.

Reply:
Done @Youngwb